rules/linux/execution_mysql_binary.toml

```
@@ -24,9 +24,7 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-sequence by host.id, process.pid with maxspan=1m
-[process where process.name == "mysql" and process.args == "-e" and process.args : ("\\!*sh", "\\!*bash", "\\!*dash", "\\!*/bin/sh", "\\!*/bin/bash", "\\!*/bin/dash")]
-[process where process.parent.name == "mysql" and process.name : ("bash", "sh", "dash")]
+process where event.type == "start" and process.name in ("bash", "sh", "dash") and process.parent.name == "mysql" and process.parent.args == "-e" and process.parent.args : ("\\!*sh", "\\!*bash", "\\!*dash", "\\!*/bin/sh", "\\!*/bin/bash", "\\!*/bin/dash") 
```

format only suggestion

```suggestion
process where event.type == "start" and process.name in ("bash", "sh", "dash") and 
  process.parent.name == "mysql" and process.parent.args == "-e" and 
  process.parent.args : ("\\!*sh", "\\!*bash", "\\!*dash", "\\!*/bin/sh", "\\!*/bin/bash", "\\!*/bin/dash") 
```